src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.vb

```diff
@@ -97,13 +97,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
             End If
 
+            Dim vbProperty = TryCast(symbol, PropertySymbol)
+            Dim countOfCustomModifiersPrecedingByRef As UShort = If(vbProperty Is Nothing, 0US, vbProperty.CountOfCustomModifiersPrecedingByRef)
+
+            If symbol.ReturnsByRef AndAlso format.MemberOptions.IncludesOption(SymbolDisplayMemberOptions.IncludeRef) Then
+                AddSpace()
+                AddKeyword(SyntaxKind.ByRefKeyword)
+
+                If countOfCustomModifiersPrecedingByRef > 0 Then
+                    AddCustomModifiersIfRequired(symbol.TypeCustomModifiers.Take(countOfCustomModifiersPrecedingByRef).AsImmutable())
```

Comment:
This code will be changed along with an introduction of a new public API, in a separate change.